### DISCUSSION TITLE
[SPDM]: Chunk NOT Supported in Responder related Query.

### DISCUSSION THREAD

**Discussion by asrivastava014** [DMTF/libspdm#2152] [General]
Error Handling for Chunking:
When chunking is not supported in the SPDM framework and there is an attempt to send large requests or responses, it is crucial to implement proper error handling. Currently, the libspdm code may not adequately handle this scenario. To address this, modifications should be made to the code to detect the lack of chunking support and return appropriate error codes or error messages. This ensures that users receive clear feedback indicating the limitation and can adjust their data transmission accordingly.

Code Details:
https://github.com/DMTF/libspdm/blob/6674aa87c5a3b6991e25e22fcd1bf44e2a111331/library/spdm_responder_lib/libspdm_rsp_receive_send.c#LL621C6-L621C48

Please provide your inputs & correct us if we are wrong.

**Comment by steven-bellock** [comment]
Check out #2141, #2153, #2154, and #2155, and see if that resolves this discussion. If not feel free to file [an issue.](https://github.com/DMTF/libspdm/issues)
